### PR TITLE
comment count for multiple article and single article testing complete

### PR DESCRIPTION
**Pull request by jjcrl**
apologies in advance  if this causes any issues for putting tickets 10 and 5 into one PR! i only did this to catch up on the tickets i have remaining. thank you

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -241,10 +241,10 @@ describe("ENDPOINT TESTING", () => {
               title: "UNCOVERED: catspiracy to bring down democracy",
               topic: "cats",
               author: "rogersop",
-              body: "Bastet walks amongst us, and the cats are taking arms!",
               created_at: "2020-08-03T13:14:00.000Z",
               votes: 0,
               article_id: 5,
+              comment_count: expect.any(String),
```
Comment (dav3rid) [inline __tests__/app.test.js]: I would like to steer you away from amending any previously passing tests to account for this. We introduced `expect.objectContaining` to account for the existence of any additional properties in future without causing previous tests to fail.
I would suggest to revert this test to how it was and to create a new test asserting only for the presence of the `comment_count`.
Also, we should be able to assert exactly how many comments this article has

File: __tests__/app.test.js [hunk 2]
```diff
@@ -313,4 +313,47 @@ describe("ENDPOINT TESTING", () => {
...
+            title: "Living in the shadow of a great man",
+            article_id: 1,
+            topic: "mitch",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 100,
+            comment_count: "11",
+          });
+        });
+    });
+  });
```
Comment (dav3rid) [inline __tests__/app.test.js:376]: A test to ensure this still works when an article has no associated comments would be a good shout too 😀

File: __tests__/app.test.js (hunk 1 above)
Comment (jjcrl) [reply __tests__/app.test.js]: - [ X ] changed to object expecting , removed comment count

File: __tests__/app.test.js (hunk 2 above)
Comment (jjcrl) [reply __tests__/app.test.js:376]: - [ X ] added test for when article has no comments

File: __tests__/app.test.js [hunk 3]
```diff
@@ -355,5 +356,22 @@ describe("ENDPOINT TESTING", () => {
...
+            created_at: '2020-10-16T05:03:00.000Z',
+            votes: 0,
+            comment_count:'0'
+          },);
```
Comment (dav3rid) [inline __tests__/app.test.js]: I'd still suggest `expect.objectContaining` here if you want to test for all the other properties too

Comment (jjcrl) [reply __tests__/app.test.js]: - [ X ]  now using expect.objectContaining

**Review by dav3rid — APPROVED** [review]

**Pull request merged**